pages/quiz/whats-on-your-mind.tsx

```diff
@@ -87,39 +86,37 @@ export const WhatsOnYourMindPage = (): JSX.Element => {
 
   return (
     <Layout>
-      <Section>
-        <HeaderComponent title="What's on your mind today?" />
-        <Navigation>
-          <LinkButton
-            textContent="skip"
-            arrow="forward"
-            onClick={() => router.push('how-are-you-feeling')}
-          />
-        </Navigation>
-        <QuestionSection>
-          <h2>Choose as many as you like</h2>
+      <HeaderComponent title="Support in Lambeth" />
```

Comment:
The header component for the quiz welcome page displays 'Quiz' and then switches to 'Support in Lambeth' for the actual questions. 

It also seems confusing for questions which are about how the user feels, what's on their mind etc. to have a heading that isn't related to the questions they are being asked. The app name would be ok, but 'Support in Lambeth' seems inconsistent with the rest of the page content.

Reply:
I think "Support in Lambeth" is the de facto title of the site at the moment - I'll make that the title throughout the quiz for now.